packages/mediator-combine-pipeline/lib/MediatorCombinePipeline.ts

```diff
@@ -27,7 +27,7 @@ export class MediatorCombinePipeline<A extends Actor<H, T, H>, H extends IAction
     // and each actor output as input to the following actor.
     let handle: H = action;
     for (const actor of testResults.map(result => result.actor)) {
-      handle = await actor.runObservable(handle);
+      handle = { ...handle, ...await actor.runObservable(handle) };
```

Comment:
Ah yes, this will be needed now.
Could we add a unit tests in `mediator-combine-pipeline` that tests this new behaviour?

Reply:
Done!